histogram.js

```diff
@@ -72,27 +75,41 @@ class HistogramElement extends HTMLElement {
     this._histo = core({
       title: this.label,
     });
-    
     this.shadowRoot.appendChild(this._histo.el);
-
     const broker = getDataBroker(this);
+
+    function toggleSVGAndLoader(svgVisibility, indicatorDisplay) {
+      const indicator = this.shadowRoot.querySelector('.loading-indicator');
+      const svgContainers = this.shadowRoot.querySelectorAll('.iobio-histogram-svg-container');
```

Comment:
You don't need to select multiple elements here and loop anymore. It will just be a single element. You should also rename the variables in here to match the new approach.